Makefile

```diff
@@ -94,9 +94,16 @@ help: ## Display this help.
 
 ##@ Development
 
+## Explanation of the statement: sed -i.bak "s/'{emptytminfo}'/{}/g" config/crd/bases/*.yaml && rm config/crd/bases/*.bak
+## This is an open issue on operator-sdk: https://github.com/operator-framework/operator-sdk/issues/6346
+## TimeMachineInfo is an optional nested struct under spec section of the database CRD. However, it does not get an empty struct value '{}'
```

Comment:
change TimeMachineInfo to DBTimeMachineInfo.

Reply:
Whoops, missed that! 
Thanks :)